core/src/main/scala/com/yahoo/maha/core/RequestModel.scala

```diff
@@ -355,6 +355,7 @@ object RequestModel extends Logging {
           val isDebugEnabled = request.isDebugEnabled
           //the zone should come from request or user time zone provider, not UTC time provider
           val timezone = request.getTimezone orElse userTimeZoneProvider.getTimeZone(request)
+          info(s"timezone is " +  timezone)
```

Comment:
The s"" is for interpolated Strings.  So timezone should be interpolated inside of that String, rather than appended.

Reply:
Also, as this calls both request and the provider, maybe which provider is being used can be named here rather than adding another log into the noop provider.

Reply:
changed.